src/components/VulnerabilityEdgePrompt.js

```diff
@@ -46,7 +39,7 @@ const VulnerabilityEdgePrompt = () => {
       setImageSetID(device?.ImageInfo?.Image?.ImageSetID);
       const params = {
         id: 'view',
```

Comment:
Nitpick: this looks obsolete.

Reply:
Weird, if I remove this line of code update page does not get opened. However, I am sure that this `id` is not used anywhere.